phpBB/phpbb/attachment/resize.php

```diff
@@ -0,0 +1,355 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\attachment;
+
+use phpbb\filesystem\filesystem_interface;
+use bantu\IniGetWrapper\IniGetWrapper;
+use FastImageSize\FastImageSize;
+
+/**
+ * Attachment resize class
+ */
+class resize
```

Comment:
i care about this deeply, but it is really not important, so feel free to ignore this, however, image_resizer or resizer would be a better name. way better.